docs/user/testing-metrics.md

```diff
@@ -6,7 +6,22 @@ These functions expose a way to inspect and validate recorded metric values with
 
 ## General test API method semantics
 
-In order to prevent issues with async calls when unit testing Glean, it is important to put the Glean SDK into testing mode by calling `Glean.enableTestingMode()`.
+In order to prevent issues with async calls when unit testing Glean, it is important to put the Glean SDK into testing mode by applying the JUnit `GleanTestRule` to your test class, as shown below:
```

Comment:
We should add that this
- puts Glean in testing mode
- ~~initializes~~
- enables uploading
- clears recorded metrics 
at the beginning of each test run.

Reply:
Added this in #214